research/src/pages/component-name-matrix.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-menu: Analysis
+menu: Overview
```

Comment:
Not sure how I feel about the name being Overview as that leads me to think of an overview of Open UI not an overview of research.

I think this belongs in some way under the analysis section. Could possibly have these be the first 2 items in the analysis menu while allowing the rest to be populated in the current manner. And you could add a border to the base of the last item to show a separation between the components & the sources.

If there is a strong desire for different menus I could see it going something like this:

<general links>

<Analysis Sources>
<Component Analysis>
<Component Proposals>

As I write that out I think we may want to change proposal to blueprint or something else as they aren't necessarily proposals. But that's another issue I'll file.

Reply:
1. if you ask me, I do have strong feeling to be able separately look at high-level pages (Component matrix and Design System list). When I implement that page, I want to check that I do not broke anything on Component matrix page, but spend some time trying to find it, even if I knew it was there. Right now you can find that page, only if you looking for it.
2. Overview is boilerplate name, since I cannot thought about better one.  More then happy to change.
3. "If there is a strong desire for different menus I could see it going something like this:". Is this sentence not fully written?

Reply:
@kant2002 @una is setting up an information architecture workshop for Open UI to inform a new navigation structure and help clarify the project in general as there has been some confusion. After that is finalized we'll land this PR taking their feedback into account. @una can you ensure that this page is included when working on the IA with the group.